utils/results/set_zip_name.py

```diff
@@ -0,0 +1,42 @@
+# If you edit this file, please consider updating bids-app-template
+
+import os
+import logging
+import subprocess as sp
+
+
+log = logging.getLogger(__name__)
+
+
+def set_zip_head(context):
+    """Set name for zipped results as
+    
+           COMMAND_name_destinationID_?.zip
+
+    where:
+       COMMAND       = the name of the main command line command
+       name          = either project, subject, or session name, which
+                       is determined by the run level
+       destinationID = the id of the destination container
+       ?             = last part added for a particular zip file
+    """
+
+
+    # Set Zip file "name": either project, subject, or session name
+    if context.gear_dict['run_level'] == 'project':
+
+        name = context.gear_dict['project_label_safe']
```

Comment:
You use gear_dict, josh uses custom_dict...maybe we should unify this somehow?

Reply:
Yes.  I think at some point Josh said to use "gear_dict".  But I was thinking even that is too long.  How about just "G" or "GG" as in "gear globals"?  I though I read somewhere that global variables should be capitalized.  This should be decided on in `gear-toolkit`.